pkg/cache/cache.go

```diff
@@ -40,6 +40,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const LayerAnnotationCacheVersion = "containerd.io/snapshot/nydus-cache-version"
```

Comment:
`LayerAnnotationCacheVersion` may need add to constant of nydus snapshotter

Reply:
It's ok to place here now.